.pyup.yml

```diff
@@ -1,5 +1,6 @@
 schedule: "every day"
 search: False
+update: insecure
 requirements:
   - requirements.txt:
       updates: security
```

Comment:
Their docs are somewhat confusing. It suggests in one place that the possible values for "update" are "all, insecure, False" and in another place it says "updates" (with an s) and has "all, security, False".

Do they really have two different spellings for the value depending on whether it's global or a specific requirements file? Do they really have two different sets of values?

It's puzzling.

Do we have any other Mozilla projects that use pyup and have it configured to just mention security updates?